lntest/harness.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
 	"github.com/lightningnetwork/lnd/lnwire"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/sync/errgroup"
```

Comment:
Cool, didn't know it before!

Reply:
It's very useful! And since we have the `golang.org/x/sync` package as a dependency already anyway, we might as well use it.